app/components/memorable_date_component.html.erb

```diff
@@ -30,17 +30,18 @@
                     maxLength: 2,
                     aria: {
                       invalid: false,
-                      describedby: [
-                        "validated-field-error-#{unique_id}",
-                        "validated-field-hint-#{unique_id}",
-                      ],
+                      describedby: "validated-field-error-#{unique_id}",
+                      labelledby: "memorable-date-month-hint-#{unique_id}",
```

Comment:
I mentioned at https://github.com/18F/identity-idp/pull/7365#discussion_r1026820921, but I might worry how a screen reader would handle the fact that we use both `<label for="...">` + `aria-labelledby` to label each field. Maybe for extra assurance, do you think it'd help to create an ID for the label and then set `aria-labelledby` to point to both the label element and the hint text?

```suggestion
                      labelledby: [
                        "memorable-date-month-label-#{unique_id}",
                        "memorable-date-month-hint-#{unique_id}",
                      ],
```

Reply:
Testing this in Safari + VoiceOver, the "Month" label is currently not read when focusing the field.

https://user-images.githubusercontent.com/1779930/203563482-4ac5df1f-20a1-4e86-ac2d-cc442920340d.mov

Reply:
ahh i think i misunderstood the comment the 1st time, updated so it has the label id. that was definitely keeping the label from being associated with the input field.